libfwupd/fwupd-security-attr-private.h

```diff
@@ -287,6 +298,31 @@ G_BEGIN_DECLS
  * Since: 1.8.0
  **/
 #define FWUPD_SECURITY_ATTR_ID_SUPPORTED_CPU "org.fwupd.hsi.SupportedCpu"
+/**
+ * FWUPD_SECURITY_ATTR_ID_PLATFORM_ROLLBACK_PROTECTION
+ *
+ * Host Security ID attribute for Rollback protection of platform firmware
```

Comment:
I think we need to define what rollback is. i.e. downgrading. Maybe we want FWUPD_SECURITY_ATTR_ID_SPI_DOWNGRADE_PROTECTION instead?

Reply:
That seems _too_ specific to me for a generic security attribute.  What if the rollback protection is at a layer that is above the payload going to the SPI chip?  Or what if it's not?  I think the attribute should be generic enough to say "this vendor has a feature for rollback back protection that would prevent installing older updates" and leave implementation details to the vendor how they do it.

Reply:
Then I think we need to clarify what the difference between downgrade and rollback is, as I'm not exactly sure myself.

Reply:
I can't add more details about the implementation used.  If having a generic attribute for rollback protection is insufficient, then I would rather just not populate it.

Reply:
> I can't add more details about the implementation used

I don't think it makes sense to add it to the spec if we can't describe it in a fair amount of detail. If it's a generic attribute then it needs enough detail to be able to say "does RISC-V do the same thing".

Reply:
OK - would you rather an AMD specific attribute then or no attribute at all?

Reply:
The former please; we can always make it generic in the future like we did with DCI, but I don't think we can ever do the opposite.

Reply:
OK; everything updated up for these comments.